testing/locust-ccengine/README.md

```diff
@@ -0,0 +1,53 @@
+# locust-ccengine
+
+Performance testing for ccEngine.
+
+:warning: **Use of this code against Creative Commons infrastructure by persons
+not employed by Creative Commons constitutes abuse.**
```

Comment:
This seems unfriendly to the community; what if we have an intern or community member testing?